doc/code/qml_dla.rst

```diff
@@ -0,0 +1,12 @@
+qml.dla
+=======
+
+(Dynamical) Lie Algebra functionality in PennyLane.
```

Comment:
This sentence is included twice in the rendered docs:
![image](https://github.com/PennyLaneAI/pennylane/assets/20772922/2692eabf-6d93-4141-a2cc-b67d2c138df7)

Once from here and once from `pennylane/dla/__init__.py`, as far as I can tell.